src/ui/Navbar.fs

```diff
@@ -4,67 +4,125 @@ open Feliz
-                        prop.children [
-                            Link [
-                                prop.text "About Us"
-                                color "foreground"
-                                prop.onClick (fun _ -> Router.navigatePath "/about")
-                                prop.style [ style.cursor "pointer" ]
-                            ]
+                        Link [
+                            prop.text "About Us"
+                            prop.onClick (fun _ -> Router.navigatePath "/about")
+                            prop.style [ style.fontSize 13; style.cursor "pointer"; style.color "white" ]
                         ]
                     ]
 
                     NavbarItem [
                         Link [
                             prop.text "Contact"
-                            color "foreground"
                             prop.onClick (fun _ -> Router.navigatePath "/contact")
-                            prop.style [ style.cursor "pointer" ]
+                            prop.style [ style.fontSize 13; style.cursor "pointer"; style.color "white" ]
                         ]
                     ]
                 ]
             ]
 
             NavbarContent [
                 justify "end"
+                prop.style [ length.px 40 |> style.gap ]
                 prop.children [
                     NavbarItem [
-                        Button [ color "primary"; prop.custom ("variant", "flat"); prop.text "Cart" ]
+                        img [
+                            prop.src <| import "default" "./assets/Cart.svg"
+                            prop.onClick (fun _ -> Router.navigatePath "/")
+                            prop.style [
+                                length.px 21 |> style.height
+                                style.cursor "pointer"
+                                (if isMenuOpen then 0 else 100) |> style.filter.invert
+                            ]
+                        ]
+                    ]
+
+                    NavbarMenuToggle [
+                        prop.ariaLabel <| if isMenuOpen then "Close menu" else "Open menu"
+                        prop.className "sm:hidden"
+                        prop.style [ (if isMenuOpen then "black" else "white") |> style.color ]
                     ]
                 ]
             ]
+
+            NavbarMenu [
+                prop.style [ style.paddingTop 0; style.backgroundColor "#F7F4F0" ]
+                menuItems
+                |> List.indexed
+                |> List.map (fun (index, item) ->
+                    NavbarMenuItem [
+                        prop.key $"{item}-{index}"
+                        prop.children [
+                            Link [
+                                color
+                                <| if index = 2 then "foreground"
+                                   else if index = menuItems.Length - 1 then "danger"
+                                   else "foreground"
```

Comment:
This seems… Odd. If you're wanting different colours for each that isn't determined by behaviour (e.g. which page is open), it should probably be added to the list instead of being hardcoded by index…

Reply:
I copied it over from NextUI's example without changing it. Good catch.